test/other/embind_tsgen.d.ts

```diff
@@ -1,4 +1,16 @@
 // TypeScript bindings for emscripten-generated code.  Automatically generated at compile time.
+declare namespace RuntimeExports {
+    let HEAPF64: any;
```

Comment:
Unfortunately these type definitions are in preamble.js so they don't make it to tsc. Hoping to fix this in another PR.